recipes/rivet/recipe.yaml

```diff
@@ -0,0 +1,133 @@
+schema_version: 1
+
+context:
+  name: rivet
+  version: 3.1.11
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://gitlab.com/hepcedar/rivet/-/archive/rivet-${{ version }}/rivet-rivet-${{ version }}.tar.gz
+  sha256: 0007e8d60d2fe78deb4481d46f4a6a2f1b6595180cb93bbfebdbd5e32fc64c0a
+
+build:
+  number: 0
+  skip: win or py>311
```

Comment:
The `python 3.12.*` is still trying to be solved, but I don't understand why.

Reply:
Need to use a match function: https://rattler.build/latest/selectors/#the-match-function